ghost/core/core/server/services/bulk-email/bulk-email-processor.js

```diff
@@ -234,6 +234,9 @@ module.exports = {
                 unsubscribe_url: postEmailSerializer.createUnsubscribeUrl(recipient.member_uuid, {newsletterUuid})
             };
 
+            // create unique urls for every recipient (for example, for feedback buttons)
+            emailData = postEmailSerializer.createUserLinks(emailData, recipient.member_uuid);
```

Comment:
@lenabaidakova Guess this will not leak anything out of flag due to checks later, but maybe this should be wrapped behind the feedback labs flag so we don't change any email data unless flag is enabled

Reply:
Done :)
https://github.com/TryGhost/Ghost/pull/15619/commits/204df614de6aa9a6b1cb08b26fafb547bc0b56b0